code-of-conduct-PT.md

```diff
@@ -67,45 +67,82 @@ Formulário online: [Formulário de denúncia de violação do Código de Condut
+- Não use informações de contato de outros usuários para fins de marketing ou divulgação, a menos que tenha obtido explicitamente sua permissão para fazê-lo.
+- Se você receber mensagens de spam de outros usuários, denuncie-os imediatamente à equipe de organização da comunidade utilizando a ferramenta mencionada na Seção 7.
+
+Nosso objetivo é manter uma comunidade livre de spam e promoção excessiva, mas entendemos que os usuários podem ocasionalmente querer compartilhar informações relevantes e valiosas com a comunidade. Se você tiver dúvidas sobre o que é ou não considerado spam, entre em contato com a equipe de moderação para obter esclarecimentos.
+
+## 9. Política anti-pirataria
+
+Nós levamos os direitos de propriedade intelectual muito a sério e não toleramos a pirataria de forma alguma. Portanto, esperamos que todos os membros cumpram as leis de direitos autorais e se abstenham de qualquer atividade que viole os direitos de propriedade intelectual de outras pessoas.
+
+As seguintes atividades são estritamente proibidas:
+
+- Fazer upload, compartilhar ou distribuir material protegido por direitos autorais sem permissão do proprietário.
+- Usar nossa plataforma para facilitar a venda ou distribuição de bens falsificados ou cópias não autorizadas de material protegido por direitos autorais.
+- Postar links ou referências a sites que oferecem conteúdo pirateado.
+- Atividades que violem os direitos de propriedade intelectual.
+
+Se recebermos uma reclamação ou aviso de suposta violação de direitos autorais, tomaremos as medidas apropriadas, que podem incluir a remoção do material infrator, a remoção da pessoa de nossa comunidade e a denúncia às autoridades.
+
+## 10. Direitos sobre o nome da comunidade e uso da marca "ITBR Canada"
+
+A marca "ITBR Canada" e qualquer logotipo ou marca comercial associados são propriedade exclusiva dos organizadores da comunidade. Qualquer uso do nome da comunidade ou da marca associada sem consentimento explícito dos organizadores é estritamente proibido.
+
+Indivíduos ou organizações que desejam usar o nome ou a marca "ITBR Canada" devem obter consentimento formal prévio dos organizadores da comunidade. Tal consentimento pode ser concedido ou negado a critério exclusivo dos organizadores.
+
+Além disso, qualquer uso do nome ou da marca da comunidade deve estar em conformidade com o código de conduta da comunidade. Os organizadores reservam-se o direito de revogar o consentimento a qualquer momento se considerarem que o uso do nome ou da marca da comunidade é prejudicial à reputação ou aos valores da comunidade.
+
+Levamos a proteção do nosso nome e marca de comunidade a sério e tomaremos medidas legais apropriadas contra qualquer indivíduo ou organização que infrinja nossos direitos de propriedade intelectual.
+
+## 8. Diretrizes á serem seguidas caso houver violação do código de conduta
```

Comment:
I believe here, instead of showing topic 8, we have to show topic 11

Reply:
Resolvido.